lib/fog/network/openstack/models/security_group.rb

```diff
@@ -22,9 +22,14 @@ def security_group_rules
         end
 
         def save
-          raise Fog::Errors::Error, 'Resaving an existing object may create a duplicate' if persisted?
           merge_attributes(service.create_security_group(attributes).body['security_group'])
-          true
+          self
+        end
+
+        def update
```

Comment:
maybe we could refactor this later to provider the standard interface of the base class create/update and save methods